Project.toml

```diff
@@ -3,18 +3,22 @@ uuid = "4c0ca9eb-093a-5379-98c5-f87ac0bbbf44"
 version = "0.18.0"
 
 [deps]
-BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
 Cairo = "159f3aea-2a34-519c-b102-8c37f9878175"
+Cairo_jll = "83423d85-b0ee-5818-9007-b63ccbeb887a"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
+GTK3_jll = "77ec8976-b24b-556a-a1bf-49a033a670a6"
+Glib_jll = "7746bdde-850d-59dc-9ae8-88ece973131d"
 Graphics = "a2bd30eb-e257-5431-a919-1863eab51364"
-Homebrew = "d9be37ee-ecc9-5288-90f1-b9ca67657a75"
 Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-WinRPM = "c17dfb99-b4f7-5aad-8812-456da1ad7187"
+adwaita_icon_theme_jll = "b437f822-2cd6-5e08-a15c-8bac984d38ee"
+gdk_pixbuf_jll = "da03df04-f53b-5353-a52f-6a8b0620ced0"
+hicolor_icon_theme_jll = "059c91fe-1bad-52ad-bddd-f7b78713c282"
```

Comment:
This should not be necessary any more, as now this is a [dependency](https://github.com/JuliaBinaryWrappers/adwaita_icon_theme_jll.jl/blob/3580e98210a11fa02bf65d8a29d01deceb5e1fd8/Project.toml) of `adwaita_icon_theme_jll`

Reply:
It is necessary, since we use its exports directly.

Reply:
Uh, right